theme.json

```diff
@@ -613,21 +613,21 @@
 			"core/pullquote": {
 				"border": {
 					"bottom": {
-						"style": "none",
-						"width": "0px"
+						"style": "solid",
```

Comment:
In order to make the borders work again and keep the layout as in the design, the `bottom`, `left`, `right` and `top` attributes should be completely removed. This way there's no border when using the block as default, and borders can be added if the block settings are updated.

Please remove the mentioned attributes, but keep the `radius` attribute.

Reply:
Hello @luminuu 

I have added my new PR.

Please review it and let me know your feedback for the same.

Thanks,